pages/ugm2023.md

```diff
@@ -34,15 +34,24 @@ style="float: left; width: 220px; height: 220px; border-radius: 20px; border: 15
 
+### Dr. Cédric Bouysset
+
+<img
+src="/public/images/Cédric_Bouysset_EXS_sm.jpg"
+title="Dr. Cédric Bouysset" alt="Dr. Cédric Bouysset"
+style="float: left; width: 220px; height: 220px; border-radius: 20px; border: 15px solid white" />
+
+Cédric Bouysset is a researcher specialized in cheminformatics, molecular modeling, and software development, currently working at Exscientia in Oxford (UK). He obtained an MSc in Cheminformatics from the University of Strasbourg and a PhD in Chemistry from the Université Côte D'Azur in France, where his research focused on studying taste perception and olfaction using a range of computational approaches. He actively maintains and contributes to open-source Python packages in computational chemistry, including MDAnalysis, which he became involved with during his participation in the Google Summer of Code program working on the interoperability between MDAnalysis and RDKit.
+
 ## Program
 A draft agenda will be announced soon.
 
 ## Registration and Abstracts
-[Registrations and abstract submissions][eventbrite] are now being accepted. 
+[Registrations and abstract submissions][eventbrite] are now being accepted. **Submit an abstract for a talk, flash talk or poster until July 7, 2023**.
 
 MDAnalysis strives to be a diverse and welcoming community for all. While the event will be free to attend, a limited number of bursaries are available to enable attendance for those facing financial barriers. If you would like to apply, please follow the prompts on the [registration form][eventbrite] to **submit your bursary application by June 23, 2023**. We will contact applicants about financial awards shortly after the deadline has passed.
 
-We welcome abstracts from all areas relevant to MDAnalysis's work, including the use and development of open source software and tools for molecular simulations data analysis. Please follow the prompts on the [registration form][eventbrite] to **submit your title and abstract before the deadline, July 7, 2023**. 
+We welcome abstracts from all areas relevant to MDAnalysis's work, including the use and development of open source software and tools for molecular simulations data analysis. Please follow the prompts on the [registration form][eventbrite] to **submit your title and abstract before the deadline, July 7, 2023**. One Outstanding Talk Travel Award will be awarded to the best talk to offset travel and accommodation costs incurred to attend the MDAnalysis UGM.
 
 ## Travel and Accommodation
 Recommended accommodation details will be provided soon.
```

Comment:
can we not update the hotel info?

Reply:
I am working on negotiating the room price down a little, so was planning to wait until that was confirmed. We need to collect names/emails from people who plan to book at the hotel under our group, so thinking about best ways to gather that info. Perhaps we add a question to the Eventbrite? Currently drafting an email to send to those already registered.

Reply:
I've added in some general hotel information for now so that we can get this posted. Can update with the specific rate after we get confirmation from the hotel.